### ISSUE TITLE
SyntaxError: unmatched ')'

### ISSUE THREAD

**Issue by TesaLazarenko** [MHProDev/PyRoxy#4]
```
  File "/usr/local/lib/python3.10/site-packages/PyRoxy/Tools/__init__.py", line 18
    rand_char: Callable[[int], str] = lambda length=16: chr(Random.rand_int(0, 1000)) for _ in range(length))
                                                                                                            ^
SyntaxError: unmatched ')'

```

**Comment by TesaLazarenko** [comment]
It would be cool if you did version tags so I could use older versions

**Comment by Igorek2112** [comment]
I think it should be like this:
rand_char: Callable[[int], str] = lambda length=16: chr(Random.rand_int(0, 1000) for _ in range(length))
after the change the code is working.

**Comment by anochuv** [comment]
More correct fix: #6

**Comment by MHProDev** [comment]
fixed
